AWS_FLB_CHERRY_PICKS

```diff
@@ -0,0 +1,5 @@
+https://github.com/krispraws/fluent-bit.git tls_net_read_fix 8d1cfeb5ba830b360fe6e1228190ed900842a3ea
+https://github.com/zhonghui12/fluent-bit.git custom-1.8.7 30fc6305695623cbc95d51df07ae185dfec8bff2
+https://github.com/matthewfala/fluent-bit.git immutable-1.8-imds-r2-cherry-picks-r2 e44f8789d605f9ab19a212b4139bc7fec8d0b530
+https://github.com/matthewfala/fluent-bit.git immutable-1.8-imds-r2-cherry-picks-r2 30b00b295db9e3ee646a0262a07ca53cf0025b58
+https://github.com/matthewfala/fluent-bit.git immutable-1.8-imds-r2-cherry-picks-r2 890915050ef2353a06c7d3e079e4d2ce821fc30b
```

Comment:
Commit 1 e44f8789: The sync network timeout
Commit 2 30b00b29:  Default to old sync network code when timeout is not used
Commit 3 89091505:  IMDS v1 fallback patch

Reply:
Added documentation directly to the cherry-pick enumeration file.